packages/server/database/migrations/20210527170505-addAdditionalRetroTemplates.ts

```diff
@@ -0,0 +1,199 @@
+import {R} from 'rethinkdb-ts'
+import getRedis from '../../utils/getRedis'
+
+const createdAt = new Date()
+const redis = getRedis()
```

Comment:
`getRedis` is going to create a connection to redis whenever this file gets imported. side effects are gross! what if up/down never get called? 
to fix that, you can put this inside the up/down function so it only creates that connection

OR, even better:

just get rid of redis calls all together. the server will flush redis when it calls `postdeploy`. you can do that in local dev, too. just call `yarn postdeploy` and it'll flush redis for you

Reply:
Makes sense! I figured it might not be necessary. I've removed the redis calls and pushed the update!